src/ParseServiceProvider.php

```diff
@@ -70,6 +70,8 @@ protected function setupParse()
 
             ParseClient::setServerURL($serverURL, $mountPath);
         }
+
+        ParseClient::setStorage(new ParseSessionStorage());
```

Comment:
We should make this configurable if it's registered or not, otherwise this breaks loading console apps, or APIs, and the like.